pkg/web/static/js/transactionAcceptPreview.js

```diff
@@ -291,4 +291,18 @@ document.body.addEventListener('htmx:configRequest', (e) => {
     data.transaction.amount = isNaN(amount) ? data.transaction.amount : amount;
     e.detail.parameters = data;
   };
-});
\ No newline at end of file
+});
+
+
+// Disable submit button to prevent multiple form submissions.
+function disableSubmitButton() {
```

Comment:
Once a user clicks the `Accept` button in the transaction accept form, they're redirected to another page. If successful, users will not be able to see the form again so that is why an enable function hasn't been included. I should note that the repair workflow will be handled in a separate story, so this may be updated then.